docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,260 @@
+                                     # be changing the history of the 
+                                     # branch remotely
+    git fetch aces
+    git checkout aces/master         # Checkout the branch that your PR is      
+                                     # based off of 
+    git checkout -b "your-branch"    # Create a “new” branch with the same
+                                     # name as the original branch
+    git cherry-pick <commit-hash>    # Cherry-pick the commits that you would like to keep
+    git push -f origin "your-branch"
+    ```
+        
+## 3. Making a Pull Request
+Once you have created a new branch and pushed an initial commit to it, you will see the option to make a pull request on GitHub. 
+Make sure that the branch you are merging into is the branch that you checked-out when you first started making changes, such as main or a release-specific branch!
+
+PRs must be reviewed according to this [checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist), and must be approved by two reviewers before it is merged. Don’t forget to think about documentation updates like the TestPlan, changelog, and developer guide updates -- these must accompany your PR.
+
+Make sure to note special rules for SQL patches. Note that changes to Raisinbread data do not need patches, because Rida manages it.
+
+**Do not** review or approve your own PR. Your PR must be reviewed and approved by two other developers before it can be merged. 
+
+**Important:** Do not merge your own PR or any other PR. After a PR has been reviewed and approved, the code is merged by Dave, the repository maintainer. It will be merged eventually. If your PR is on a release branch and has been ready to merge for a few days, you can add a comment that it is ready to merge. 
+
+It’s your responsibility to be on top of your PRs, and any Issues that need follow-up.  This includes answering questions and comments, acknowledging and making change requests, and requesting re-review after you’ve addressed someone’s change requests. 
+
+This is important for the Team’s productivity -- it prevents PRs from lagging and prevents confusion as to why a PR has stalled or if you’ve seen someone’s comments.
+When issuing a PR, you can request a specific reviewer (this is not the same as assigning a PR to someone).  There is usually an obvious reviewer for a PR.  
+This is especially important in Release-Test/Fix cycles when we want PRs to be reviewed and resolved quickly.
+
+### PR Formatting
+Be sure to observe and follow team conventions for how PRs should be named and described. 
+
+**Title:** [module_name] Quick description   
+
+**[Contents](../../../.github/PULL_REQUEST_TEMPLATE.md) of initial comment:** Should include a brief summary of the changes and rationale, links to related issues or PRs, and testing instructions. 
+
+**Linking issues:** Use GitHub keywords like `Fixes` or `Resolves` so that the issue is closed automatically when the PR is merged.
```

Comment:
add : do not use these key words if you are only addressing one point raised in the issue.  this will close the entire issue and leave unresolved bugs.